README.md

```diff
@@ -1,2 +1,37 @@
-# AR6-WG1-Data-Compilation
-Compilation of data from AR6 WG1
+# Data Compilation & Figures for the IPCC AR6 WG1
+
+© 2021 Zebedee Nicholls & Daniel Huppmann; licensed under the [BSD 3-Clause License](LICENSE).
```

Comment:
As above re copyright to contributors rather than us

Reply:
Added an explicit list of contributors

Reply:
👍